manifests/bosh/bosh.yml

```diff
@@ -61,101 +64,119 @@ instance_groups:
   - name: (( grab params.bosh_network ))
     static_ips:
       - (( grab params.static_ip ))
-
-  properties:
-    env:
-      http_proxy:  (( grab params.http_proxy  || "" ))
-      https_proxy: (( grab params.https_proxy || "" ))
-      no_proxy:    (( grab params.no_proxy    || "" ))
-
-    nats:
-      address:  (( grab params.static_ip ))
-      user:     natsuser
-      password: (( vault meta.vault "/nats:password" ))
-      tls:
-        ca: (( vault meta.vault "/nats/ca:certificate" ))
-        client_ca:
-          certificate: (( vault meta.vault "/nats/ca:certificate" ))
-          private_key: (( vault meta.vault "/nats/ca:key" ))
-        server:
-          certificate: (( vault meta.vault "/nats/server:certificate" ))
-          private_key: (( vault meta.vault "/nats/server:key" ))
+  jobs:
+  - name: bpm
```

Comment:
While we're in here making changes, can we stick to the indented list formatting of YAML uniformly?

i.e.:

```
key:
  - list item
```

instead of:

```
key:
- list item
```
?